web/packages/teleport/src/Users/useUsers.ts

```diff
@@ -82,11 +83,21 @@ export default function useUsers({
     });
   }
 
-  function onCreate(u: User) {
+  async function onCreate(u: User) {
+    const webauthnResponse = await auth.getWebauthnResponseForAdminAction(
+      'CreateUser',
```

Comment:
Similar to the other PR, I think we should have `AdminAction.CREATE_USER` or something